Automattic/sensei
# Improvements for Sensei Help section and Home.

Fixes #5901 

### Changes proposed in this Pull Request
* This basically includes all the feedback from Renatho to #5895
  * Extract Help section css to separate file.
  * Update lock icon with existing `assets/icons/lock.svg` which is more similar to the design than the one currently used.
  * Created a generic `<Link>` component to be used in Sensei Home that will display an "external" icon if the url is external.
  * Fix styles for Help entries for smaller screens.
* Bonus: Error handling as suggested by Renatho too on slack.

### Testing instructions
* Checkout to the branch.
* Go to Sensei Home and check that everything looks nice.
* To test the error handling you might want to add `throw new Error("Oops!");` in the try/catch in `fetchAndSetData` function.

### Screenshot / Video
#### Error while fetching data
<img width="1391" alt="image" src="https://user-images.githubusercontent.com/799065/195885990-cff55cff-b17f-4e74-bfed-e32396fe7011.png">

#### Get Help section (big screen)
<img width="610" alt="image" src="https://user-images.githubusercontent.com/799065/195886054-02e1dd1e-1863-41a4-92e5-113ef6e446f8.png">

#### Get Help section (small screen)
<img width="340" alt="image" src="https://user-images.githubusercontent.com/799065/195886183-604b58ee-6b9d-40b4-b2d5-3328d13238c4.png">
